modules/client/test/integration/tokenVoting-client/methods.test.ts

```diff
@@ -348,7 +348,7 @@ describe("Token Voting Client", () => {
         const ctxPlugin = ContextPlugin.fromContext(ctx);
         const client = new TokenVotingClient(ctxPlugin);
         const limit = 5;
-        const status = ProposalStatus.EXECUTED;
+        const status = ProposalStatus.DEFEATED;
```

Comment:
Curious to understand why this test is changed

Reply:
There are no executed proposals in subgraph

Reply:
There are somee scripts, but I belive they are not automated, but yes it will be super cool if we can populate with test data each time we launch the contracts

Reply:
I would like to mention this again. I suggest we start mocking this and don't rely on external services to have the correct data so that tests succeed.
As mentioned in multiple dev guides: unit tests shouldn't have external dependencies. External dependencies should be either mocked or stubbed to get reproducible test results

I don't see the point of wasting our goerliETH to generate dummy data so that the SDK has tests passing

Reply:
@mathewmeconry let's get it done on next sprint (APP-1643) :+1: